pkgs/by-name/te/terminal-toys/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: rec {
+  pname = "terminal-toys";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Seebass22";
+    repo = "terminal-toys";
+    tag = "v${version}";
```

If using `finalAttrs`, prefer

```suggestion
    tag = "v${finalAttrs.version}";
```